fiftyone/operators/server.py

```diff
@@ -191,10 +192,37 @@ async def post(self, request: Request, data: dict) -> dict:
         return result.to_json() if result else {}
 
 
+class ResolveExecutionOptions(HTTPEndpoint):
+    @route
+    async def post(self, request: Request, data: dict) -> dict:
+        dataset_name = resolve_dataset_name(data)
+        dataset_ids = [dataset_name]
+        operator_uri = data.get("operator_uri", None)
+        if operator_uri is None:
+            raise ValueError("Operator URI must be provided")
+
+        registry = await _get_operator_registry_for_route(
```

Comment:
For consistency. May disregard
```suggestion
        registry = await PermissionedOperatorRegistry.from_exec_request(
            request, dataset_ids=dataset_ids
        )
```